keps/sig-api-machinery/3962-mutating-admission-policies/README.md

```diff
@@ -487,6 +481,24 @@ mutations:
 
 ```
 
+For granular list removal, an use case would be removing an entire container with an envvar named `remove-this-container`.
+```yaml
+mutations:
+  - mutation: >
+      Object{
+        spec: Object.spec{
+                containers: [Object.spec.containers{
+                       envvar: "remove-this-container",
```

Comment:
Note that envvar is not a key of containers.

Reply:
Is this intended or a typo? Would we expect this to remove all containers with an envar with an element "remove-this-container"? Is this meant as a convenience for the user to not have to collect the keys off the items they first want to remove?

Reply:
I wouldn't expect this to be supported as-written. @cici37 was this intended to show removal of a listMap item by key?  Should we update this to show the use of a valid key (since envvar is not a container key field)?

Reply:
This originated from a question from me: https://github.com/kubernetes/enhancements/pull/4399/files#r1450782169

I am interested in removing all containers based on some check of said containers that is not its name.  How would I achieve that?

Reply:
Sorry for the confusion. I have updated the example. The filter should be used to specify a item removal with certain criteria like examples showing above. We initially considered to have an `operation: deletion` field added to further clear the operation and reduce the confusion. Happy to hear others' thought on this :)